reference/interbit/createChainsFromManifest.js

```diff
@@ -0,0 +1,3 @@
+const { createChains: { createChainsFromManifest } } = require('interbit')
+
+const { ??? } = await createChainsFromManifest(cli, manifest)
```

Comment:
The promise this function returns resolves to nothing. The context from `createChainsFromConfig` is generated dynamically during the function operation whereas when calling `createChainsFromManifest` it is already present in the manifest passed into params so unnecessary to return.